website/migrations/0091_bid.py

```diff
@@ -35,6 +35,7 @@ class Migration(migrations.Migration):
                 ("amount", models.IntegerField()),
                 ("status", models.CharField(default="Open", max_length=10)),
                 ("pr_link", models.URLField(blank=True, null=True)),
+                ("bch_address", models.CharField(blank=True, null=True, max_length=45)),
             ],
```

Comment:
okay and how would we know whose bid was selected without username >

Reply:
Okay , And Also we dont need to complicate anything we will know it by the pull request

Reply:
But the main motive of fetching old bid is to make sure user makes a bid less than the last bid

Reply:
bid_code = str(uuid.uuid4())
we can use something like this or random function to make a code and we can fetch this code to verify the bid , what do you think ?

Reply:
Okay I will change that and and are we paying the user before the work is done  or after the work is done ?

Reply:
How will we implement proof of funds in the form ?

Reply:
> > bid_code = str(uuid.uuid4())
> > we can use something like this or random function to make a code and we can fetch this code to verify the bid , what do you think ?
> 
> No need to. The presence of the bid in the table is good enough verification please see this for more details of how it works #2092

We already have a function which returns a image with value of bid we can use that and add other informations in it too

Reply:
> > > > bid_code = str(uuid.uuid4())
> > 
> > 
> > > > we can use something like this or random function to make a code and we can fetch this code to verify the bid , what do you think ?
> > 
> > 
> > > 
> > 
> > 
> > > No need to. The presence of the bid in the table is good enough verification please see this for more details of how it works #2092
> > 
> > 
> > We already have a function which returns a image with value of bid we can use that and add other informations in it too
> 
> Sounds good

So, Should I give user the link to it (the bid info picture) as a code snippet to verify in github comments

Reply:
> > How will we implement proof of funds in the form ?
> 
> I think I each issue will have its own page with bid history. On that page will be a button [accept bid] and that will show a form to enter the proof of funds bch address, that will also close the bidding and will give 24 hours for the pull request. Otherwise it will open the bidding again if no pull request was made.

How about having a form for admins where they can either paste an issue link or choose from a dropdown list of unique issues? Once they select an issue, the form displays all the bids placed on that issue, along with an option to select a bid.

Reply:
Okay ! Sounds good ! , thank you